README.md

```diff
@@ -1,11 +1,109 @@
-# Hacktoberfest2020
+# Hacktober Fest 2020
+🙅🏼‍♂️Hacktoberfest is open to everyone in our global community. Whether you’re new to development, a student, long-time contributor, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skills levels are encouraged to complete the challenge.🙅🏼‍♂️
 
-- Use this repository to create your PRs during Hacktoberfest event.
-- Be creative while adding your code
-- Code should be working perfectly and add a screeenshot/gif to support your statement.
-- Add comments to explain hard parts but don't add unnecessay comments.
-- Have fun coding!
 
-# Suggestions or improvements?
-Make an issue of the same and we will discuss it!
 
```

Comment:
Add the language batches before instructions. Preferred badges: 
- Python,c, c++c, java, HTML, git